### ISSUE TITLE
MapStore2 binary not working with JDK 11 

### ISSUE THREAD

**Issue by randomorder** [geosolutions-it/MapStore2#7600]
## Description
[MapStore2 binary](https://github.com/geosolutions-it/MapStore2/releases/download/v2021.02.00/mapstore2-2021.02.00-bin.zip) is not starting with JDK version 11

## How to reproduce
Download and extract the zip file and try to start mapstore with JAVA_HOME set to a version 11 JDK

*Expected Result*
MapStore running

*Current Result*
MapStore not running

- [x] Not browser related

## Other useful information
```
-Djava.endorsed.dirs=/home/alpa/mapstore2/endorsed is not supported. Endorsed standards and standalone APIs
in modular form will be supported via the concept of upgradeable modules.
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

**Comment by FP-Geosolutions** [comment]
The `java.endorsed.dir` parameter was deprecated by version 9 of java and removed by version 11, here the reference https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-8E83E51A-88A3-4E9A-8E2A-66E1D66A966C
to fix it you have to remove the parameter from` catalina.sh/bat` or upgrade the tomcat version

**Comment by offtherailz** [comment]
On master this has been fixed in this commit: 
https://github.com/geosolutions-it/MapStore2/commit/1760520f5956823c9b5da23994ffcae66c4ae346

@randomorder on stable branch this commit can not be backported because it contains a lot of breaking changes. So if we want to make it work with Java 11, we need to manually change tomcat version (checking the `/release/ `folder content).

**Comment by offtherailz** [comment]
I don't know if it makes sense to update the whole tomcat for the stable release when the next year we will have 2 more stable releases already fixed to work in java 11.
Maybe changing the startup script to make it work can be a good compromise.

**Comment by offtherailz** [comment]
@randomorder Binary should work with it's own embedded JRE Imho.
 For this reason, I think that this will be fixed by #7513 
Please @FP-Geosolutions let me know if it makes sense.

**Comment by offtherailz** [comment]
Going to close this because is fixed on master and in favor of #7513
